src/Http/Controllers/Auth/LoginController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers\Auth;
+
+use Blumilk\Meetup\Core\Http\Controllers\Controller;
+use Blumilk\Meetup\Core\Http\Requests\LoginUserRequest;
+use Blumilk\Meetup\Core\Models\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+
+class LoginController extends Controller
+{
+    public function formPage()
+    {
+        return view("user.login");
+    }
+    public function login(LoginUserRequest $request)
+    {
+        $validated = $request->validated();
+        $user = User::where("email", $validated["email"])->first();
+        if (!$user || !Hash::check($validated["password"], $user["password"])) {
+            return response("Bad credential", 404);
```

Comment:
```suggestion
            return response("Bad credentials", 404);
```

Reply:
I think 404 code is not good HTTP code for bad credentials (401 would be ok) and I would throw in that case exception (Laravel exception) like that 
`throw new AuthenticationException()`